src/compress/zip/file_info.cr

```diff
@@ -5,7 +5,7 @@ module Compress::Zip::FileInfo
 
   DEFLATE_END_SIGNATURE = Bytes[80, 75, 7, 8, read_only: true]
 
-  property version : UInt16 = Zip::VERSION
+  property version = 20_u8
```

Comment:
1. Why remove the relation to the constant?
2. Changing the type on this public property would be a breaking change.

Reply:
1. Zip versions are confusing. The 2 versions should **not** be tied together.

There's the software `VERSION` and the extraction `version`.  Extraction version should probably be `10_u8` to match unzip and conditionally incremented based on features used (examples: encryption, bzip2).
I need to read unzip more to make sure that sort of change is safe and beneficial.

What should `VERSION` be?  IDFK.  It's possibly only informative.  I haven't read enough zip implementations yet to figure out if the value is used.

Reply:
2. This sort of property is only useful if you are implementing encryption, compression (bzip2/zstd/etc) other optional features or possibly want to print the values.  Implementing encryption or compression would require modifying several crystal zip methods so I think you'd possibly have to update PR's for those but not worry about shards.

#7236 would need minimal modification.

It should possibly be `:nodoc:` as it's only useful during zip/unzip.

I doubt anyone else uses it.  

The zip spec is odd in describing the version as a `UInt16` which maybe it was before 1992.  1 byte is the version and the other the operating system or file system identifier.